applications/welcome/models/proc_harness.py

```diff
@@ -1,26 +0,0 @@
-#########################################################################
-## Define API for interacting with procedure table
-## These methods will be used in the controller, called by the code
-## that actually performs the sync to interact with the tables
-##
-## Additionally, these methods will be used by the editor team to
-## perform saves and stable saves
-#########################################################################
-
-from gluon import current
-
-proc_table = current.db["procedures"]
-
-def save(procedure_entry): #not exactly sure what this method needs
-    pass
```

Comment:
These are in the welcome app. Not in client. These were added to welcome through some other app or probably removed during their merge with master.